store/pruning/manager.go

```diff
@@ -78,15 +77,15 @@ func (m *Manager) HandleSnapshotHeight(height int64) {
 	defer m.pruneSnapshotHeightsMx.Unlock()
 
 	m.logger.Debug("HandleSnapshotHeight", "height", height)
-	m.pruneSnapshotHeights = append(m.pruneSnapshotHeights, height)
-	sort.Slice(m.pruneSnapshotHeights, func(i, j int) bool { return m.pruneSnapshotHeights[i] < m.pruneSnapshotHeights[j] })
-	k := 1
-	for ; k < len(m.pruneSnapshotHeights); k++ {
-		if m.pruneSnapshotHeights[k] != m.pruneSnapshotHeights[k-1]+int64(m.snapshotInterval) {
-			break
-		}
+
+	if len(m.pruneSnapshotHeights) == 0 {
+		// If there are no snapshot heights, add the current height.
+		m.pruneSnapshotHeights = append(m.pruneSnapshotHeights, height)
```

Comment:
`NewManager` constructor initiates `pruneSnapshotHeights` with `[]int64{0}`. would that mean this code block would never get executed?

Reply:
Yes, it should never be executed in normal cases. just add the condition to prevent any unexpected cases.